### ISSUE TITLE
new skinchanger (with cslolmenager)

### ISSUE THREAD

**Issue by T9Jungler** [R3nzTheCodeGOD/R3nzSkin#1162]
![Screenshot_1](https://github.com/R3nzTheCodeGOD/R3nzSkin/assets/142430381/755fa7b0-f913-4835-8f2b-1c660f6a2076)
i got this skins for cslolmenager 
wait for full relese app to get all skins like this i will give dc server from guy but i got only few skin bcs i dont have close beta i got this from friend with tepmorary role on dc

here is dc:  https://discord.gg/ainzskin

**Comment by kishou234** [comment]
you mind sending me those files on discord my @ is 0_edis

**Comment by ComplexQa** [comment]
So how does this work exactly, canu release a tutorial or somekind of explanation or is not done yet? many thanks

**Comment by T9Jungler** [comment]
> So how does this work exactly, canu release a tutorial or somekind of explanation or is not done yet? many thanks

![Screenshot_2](https://github.com/R3nzTheCodeGOD/R3nzSkin/assets/142430381/1f747fe0-3730-492a-a8b2-8e34e7a1e170)
![Uploading Screenshot_4.png…]()

**Comment by T9Jungler** [comment]
![Uploading Screenshot_4.png…]()

**Comment by T9Jungler** [comment]
![Screenshot_5](https://github.com/R3nzTheCodeGOD/R3nzSkin/assets/142430381/46b59a0d-b767-4811-838b-b2b561c66a66)

**Comment by ComplexQa** [comment]
yes thank you for pictures of the program, but how do i download it

**Comment by T9Jungler** [comment]
> yes thank you for pictures of the program, but how do i download it

there is only private version for now but i thing full version will be released soon

**Comment by T9Jungler** [comment]
just join dc and follow updates

**Comment by ComplexQa** [comment]
ah ok thanks

**Comment by SomethingSomethingDarkS** [comment]
discord expired link, is there a new one?

**Comment by Vickeblanca** [comment]
discord link expired please send new one.
